src/jitlayers.cpp

```diff
@@ -1137,24 +1144,21 @@ void jl_dump_native(const char *bc_fname, const char *unopt_bc_fname, const char
         }
     }
 
-    ValueToValueMapTy VMap;
-    Module *clone = shadow_output;
```

Comment:
Ah, right nevermind; we've dropped support for versions of LLVM where `jl_add_to_shadow` wasn't already making the clone.